package.json

```diff
@@ -50,14 +50,14 @@
     "@rollup/plugin-typescript": "^9.0.2",
     "@testing-library/react": "^12.1.2",
     "@types/jest": "^27.5.2",
-    "@types/styled-components": "^5.1.26",
```

Comment:
styled-components v6 has types included and doesn't need a separate types package anymore :)